example/notebook/installation_register_and_convert_model.ipynb

```diff
@@ -0,0 +1,481 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.7-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.7.7 64-bit ('modelci': conda)",
+   "metadata": {
+    "interpreter": {
+     "hash": "c50a05bb497594c0bc31527bf55a05592e27b3d7a0a143acd08dceef0e91886f"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "# A getting started tutorial of ModelCI installation, Register and convert Model\n",
+    "\n",
+    "This is a getting started tutorial for those who are new to ML-ModelCI, including the following steps:\n",
+    "\n",
+    "- Setting up environment\n",
+    "- Starting ModelCI service.\n",
+    "- Loading a pretrained resnet50 model from torchvision.\n",
+    "- Register this model into modelHub.\n",
+    "\n",
+    "  The converter will automatically convert this PyTorch format model into 2 model : a Open Neural Network eXchange (ONNX) format model and a TorchScipt format model\n",
+    "\n",
+    "- Converting the ONNX format model into TensorRT format manually.\n",
+    "- Stopping ModelCI service.\n",
+    "\n",
+    "## 1. Installation\n",
+    "\n",
+    "Make sure [conda]() and [docker]() are installed before this tutorial, and docker service has already started.\n",
+    "\n",
+    "Besides, you should create modelci conda environment based on yml file \n",
+    "\n",
+    "```shell\n",
+    "# create conda environment based on given yml file\n",
+    "wget -q https://raw.githubusercontent.com/cap-ntu/ML-Model-CI/master/environment.yml\n",
```

Comment:
jupyter notebook can execute the system command with `!`, you can add those commands in the cells.

Reply:
And I think conda environment are only for developers, if we use pip installation, we don't need that anymore. Am I right? @YuanmingLeee

Reply:
> jupyter notebook can execute the system command with `!`, you can add those commands in the cells.

It's because Conda environment management is difficult if executed in jupyter notebook, especially switch to a certain conda env, if we could skip conda environment, it will be much easier.

By the way, I want to make sure if it's necessary for us to include installation setup in this jupyter notebook,  for there will be long output if we install pip packages in the notebook, which may lead to poor readability. 

If it's necessary, I will try to find ways to eliminate useless installation output.